api/download.py

```diff
@@ -47,7 +47,7 @@ def _append_targets(self, targets, cont_name, container, prefix, total_size, tot
                 if filtered:
                     continue
             if optional or not f.get('optional', False):
-                filepath = os.path.join(data_path, util.path_from_uuid(f.get('_id', '')))
+                filepath = os.path.join(data_path, 'v1', util.path_from_uuid(f.get('_id', '')))
```

Comment:
Would it be possible to move all these path joins to a helper function? There's 6+ instances of this logic, including the `v1` string.